apps/distgo/config/config.go

```diff
@@ -256,11 +256,23 @@ type DockerDep struct {
 	TargetFile string `yaml:"target-file" json:"target-file"`
 }
 
+type DockerImageInfo struct {
+	Type string      `yaml:"type" json:"type"`
+	Data interface{} `yaml:"data" json:"data"`
+}
+
 type DockerImage struct {
-	Repository string      `yaml:"repository" json:"repository"`
-	Tag        string      `yaml:"tag" json:"tag"`
-	ContextDir string      `yaml:"context-dir" json:"context-dir"`
-	Deps       []DockerDep `yaml:"dependencies" json:"dependencies"`
+	Repository string          `yaml:"repository" json:"repository"`
+	Tag        string          `yaml:"tag" json:"tag"`
+	ContextDir string          `yaml:"context-dir" json:"context-dir"`
+	Deps       []DockerDep     `yaml:"dependencies" json:"dependencies"`
+	Info       DockerImageInfo `yaml:"info" json:"info"`
+}
+
+type SLSImageInfo struct {
```

Comment:
`SLSDockerImageInfo` would probably be a better name (wouldn't feel as strongly about this if this were in a `Docker` package or something, but since it's in a general `config` package I think it makes more sense to be explicit about the naming)